pennylane/ops/op_math/controlled_decompositions.py

```diff
@@ -152,78 +151,79 @@ def ctrl_decomp_zyz(target_operation: Operator, control_wires: Wires):
 
     .. code-block:: python
 
-        dev = qml.device("default.qubit", wires=3)
+        dev = qml.device("default.qubit", wires=2)
 
         @qml.qnode(dev)
         def expected_circuit(op):
             qml.Hadamard(wires=0)
-            qml.Hadamard(wires=1)
-            qml.ctrl(op, [0,1])
+            qml.ctrl(op, [0])
```

Comment:
Why are we changing the example to drop a control wire?

Reply:
I'm not sure I understand why this function no longer works with multiple control wires.